### PR TITLE
Add note for the status bar icon

### PR DESCRIPTION
**Pull request by Cassie205**

### CODE REVIEW

**Review by dkocher — CHANGES_REQUESTED** [review]
Add note for Windows as well for cases icon is not visible.

File: mountainduck/interface.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ Right-click on the taskbar and select *Taskbar setting → Notification area →
 
 `````
 
+```{tip}
+Using macOS, make sure that there is enought screen real estate left on the right side of the system status bar otherwise the Mountain Duck icon might not be visible.
```
Comment (dkocher) [inline mountainduck/interface.md:36]: Typo `enought`

File: mountainduck/interface.md [hunk 2]
```diff
@@ -32,6 +32,10 @@ Right-click on the taskbar and select *Taskbar setting → Notification area →
...
+```
+
```
Comment (dkocher) [inline mountainduck/interface.md:38]:
```suggestion
```{admonition} macOS only
:class: tip

Make sure there is enough space left on right side of the notch or application menu items to make sure the _Mountain Duck_ is visible in the the system status bar
```
```

Comment (tatze96) [reply mountainduck/interface.md:38]: ensure to add the dot at the end of the sentence and add the closing of the admonition below the sentence.

**Comment by Cassie205** [comment]
> Add note for Windows as well for cases icon is not visible.

I'm not aware of any Windows-related cases with not visible icons

**Review by dkocher — APPROVED** [review]

**Pull request merged**